src/app/modules/registration/pages/snow/avalanche-activity/avalanche-activity-modal/avalanche-activity-modal.page.html

```diff
@@ -1,59 +1,57 @@
 <ion-header>
-  <ion-toolbar appHeaderColor mode="ios">
-    <ion-buttons slot="start">
-      <ion-button (click)="cancel()">{{ 'DIALOGS.CANCEL' | translate }}</ion-button>
+  <ion-toolbar appHeaderColor mode='ios'>
+    <ion-buttons slot='start'>
+      <ion-button (click)='cancel()'>{{ 'DIALOGS.CANCEL' | translate }}</ion-button>
     </ion-buttons>
     <ion-title>{{'REGISTRATION.SNOW.AVALANCHE_ACTIVITY.TITLE' | translate}}</ion-title>
   </ion-toolbar>
 </ion-header>
 
-<ion-content *ngIf="avalancheActivityCopy">
-  <ion-list lines="full">
+<ion-content *ngIf='avalancheActivityCopy'>
+  <ion-list lines='full'>
     <ion-item>
       <ion-label>{{'REGISTRATION.SNOW.AVALANCHE_ACTIVITY.NO_AVALANCHE_ACTIVITY' | translate}}</ion-label>
-      <ion-checkbox mode="md" slot="start" [(ngModel)]="noAvalancheActivity"></ion-checkbox>
+      <ion-checkbox mode='md' slot='start' [(ngModel)]='noAvalancheActivity'></ion-checkbox>
     </ion-item>
     <ion-item [ngClass]="{'hasWarning': dateIsDifferentThanObsTime}">
-      <ion-label color="medium" position="stacked" class="ion-text-uppercase">{{
+      <ion-label color='medium' position='stacked' class='ion-text-uppercase'>{{
         'DATE' | translate }}</ion-label>
-      <ion-datetime color="medium" monthShortNames="{{ 'MONTHS.SHORT_LIST' | translate }}"
-        doneText="{{'DIALOGS.OK' | translate}}" cancelText="{{'DIALOGS.CANCEL' | translate}}"
-        display-format="D. MMM, YYYY" [max]="maxDate" [(ngModel)]="startDate">
-      </ion-datetime>
+      <app-datetime-picker [(localDate)]='startDate' [maxDate]='maxDate'></app-datetime-picker>
```

Comment:
Viser tidspunkt 00:00. Tror ikke tidspunkt er relevant her.
Sjekk selv under  Snø / Skredaktivitet /Dato

Reply:
La inn ny formattering her, og la til support for "format", slik at vi ikke trenger å vise tid ([https://ionicframework.com/docs/api/datetime#date-selection](https://ionicframework.com/docs/api/datetime#date-selection))